suministrospr/suministros/management/commands/import_data.py

```diff
@@ -16,31 +17,68 @@ def normalize(self, text):
     def add_arguments(self, parser):
         parser.add_argument("target_folder", type=str)
 
+    def get_suministro(self, slug):
+        try:
+            return Suministro.objects.get(slug=slug)
+        except Suministro.DoesNotExist:
+            return None
+
+    def get_import_data(self, data):
+        municipality = self.normalize(data.get("municipio", ""))
+        title = data.get("title", "")
+        slug = (
```

Comment:
@froi this is an important part of making this change work. As mentioned in https://github.com/Code4PuertoRico/suministrospr/issues/34#issuecomment-574405190, we can use the original slugs (example: https://suministrospr.com/sectores/alturas-de-belgica/ - **alturas-de-belgica**) as the idempotency key.

In addition, this will help us not break as many existing links as possible.